cmd/tools/rest/client.go

```diff
@@ -194,7 +199,7 @@ func (c *Client) GetRest(request string) ([]byte, error) {
 	}
 	c.request.Header.Set("accept", "application/json")
 	if c.username != "" {
-		c.request.SetBasicAuth(c.username, auth.Get().Password())
+		c.request.SetBasicAuth(c.username, c.auth.Password())
```

Comment:
few places we use c.Password() and others as c.auth.Password(). 
May be we should use same for all.

Reply:
yes! good catch. we should pass all through auth